src/System.Diagnostics.Tracing/src/System/Diagnostics/Tracing/EventProviderBase.cs

```diff
@@ -1,1905 +0,0 @@
-    /// 
-    /// In addition, when new eventSources are created, the 'OnEventSourceCreate' method is called. The
-    /// invariant associated with this callback is that every eventSource gets exactly one
-    /// 'OnEventSourceCreate' call for ever eventSource that can potentially send it log messages. In
-    /// particular when a EventListener is created, typically a series of OnEventSourceCreate' calls are
-    /// made to notify the new dispatcher of all the eventSources that existed before the EventListener was
-    /// created.
-    /// 
-    /// </summary>
-    public abstract class EventListener : IDisposable
-    {
-        /// <summary>
-        /// Create a new EventListener in which all events start off truned off (use EnableEvents to turn
-        /// them on).  
-        /// </summary>
-        protected EventListener()
-        {
-            lock (EventListenersLock)
-            {
-                // Find all existing eventSources call OnEventSourceCreated to 'catchup'
-                foreach (WeakReference eventSourceRef in s_EventSources)
-                {
-                    EventSource eventSource = eventSourceRef.Target as EventSource;
-                    if (eventSource != null)
-                        eventSource.AddListener(this);
-                }
-
-                // Add to list of listeners in the system 
-                this.m_Next = s_Listeners;
-                s_Listeners = this;
-
-                Validate();
-            }
-        }
-        /// <summary>
-        /// Dispose should be called when the EventListener no longer desires 'OnEvent*' callbacks. Because
-        /// there is an internal list of strong references to all EventListeners, calling 'Displose' directly
-        /// is the only way to actually make the listen die. Thus it is important that users of EventListener
-        /// call Dispose when they are done with their logging.
-        /// </summary>
-        public virtual void Dispose()
-        {
-            lock (EventListenersLock)
-            {
-                Contract.Assert(s_Listeners != null);
-                if (s_Listeners != null)
-                {
-                    if (this == s_Listeners)
-                    {
-                        EventListener cur = s_Listeners;
-                        s_Listeners = this.m_Next;
-                        RemoveReferencesToListenerInEventSources(cur);
-                    }
-                    else
-                    {
-                        // Find 'this' from the s_Listeners linked list.  
-                        EventListener prev = s_Listeners;
-                        for (; ;)
-                        {
-                            EventListener cur = prev.m_Next;
-                            if (cur == null)
-                                break;
-                            if (cur == this)
-                            {
-                                // Found our Listener,
```

Comment:
Just making sure.  EventProviderBase.cs was the old name for EventSource.cs   I like the idea of harmonizing on the new name, but I also want to make sure that this was what was intended and that no other changes 'snuck in' because of the file name change.